### PR TITLE
zh_CN: translation js_tutorial_training_linear_regression.md 

### PR DESCRIPTION
**Pull request by PeterJaq**

### CODE REVIEW

**Comment by lamberta** [comment]
FYI @JayYip, @yantaozhao, @loveunk, @Wind2esg

File: site/zh-cn/js/tutorials/training/linear_regression.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# 从二维数据进行预测
+
+在这个教程中你将使用一组汽车的数字数据集训练一个模型并且预测。
+
+本练习将演示训练许多不同类型模型的常用步骤，但将使用小数据集和简单（浅）模型。主要目的是帮助您熟悉TensorFlow.js训练模型的基本术语、概念和语法，并为进一步的探索和学习提供一个垫脚石。
+ The primary aim is to help you get familiar with the basic terminology, concepts and syntax around training models with TensorFlow.js and provide a stepping stone for further exploration and learning.
```
Comment (lamberta) [inline site/zh-cn/js/tutorials/training/linear_regression.md]: Probably meant to remove this 😉

Comment (loveunk) [reply site/zh-cn/js/tutorials/training/linear_regression.md]: @lamberta  To remove this? I didn't get it. To remove "this document" or the English part?

Comment (lamberta) [reply site/zh-cn/js/tutorials/training/linear_regression.md]: @loveunk Remove the English part since the translation is above (I think?)

**Pull request merged**